src/libraries/Common/src/Interop/Windows/Kernel32/Interop.REPARSE_DATA_BUFFER.cs

```diff
@@ -38,5 +38,14 @@ internal struct MountPointReparseBuffer
             public ushort PrintNameOffset;
             public ushort PrintNameLength;
         }
+
+        [StructLayout(LayoutKind.Sequential)]
+        internal struct AppExecLinkReparseBuffer
+        {
+            public uint ReparseTag;
+            public ushort ReparseDataLength;
+            public ushort Reserved;
+            public uint StringCount;
+        }
```

Comment:
It is not related the PR but I wonder why do such structs returned from P/Invokes would not be _readonly_ structs?